src/test/java/com/spotify/docker/client/DockerHostTest.java

```diff
@@ -151,7 +151,7 @@ public void testFromEnv() throws Exception {
     final String dockerHostEnvVar = DockerHost.defaultDockerEndpoint();
     final boolean isUnixSocket = dockerHostEnvVar.startsWith("unix://");
     final URI dockerHostUri = new URI(dockerHostEnvVar);
-    final String dockerCertPathEnvVar = DockerHost.defaultCertPath();
+    final String dockerCertPathEnvVar = null;
```

Comment:
later in this test, this value is compared to `isNullOrEmpty()` - which is always true - and then another value is compared to this one. It would be simpler to just get rid of the reference and test the other value against `isNull()` or whatever the correct matcher is